custom/aliases/example.aliases.sh

```diff
@@ -1,2 +0,0 @@
-# Add your own custom alias in the custom/aliases directory. Aliases placed
```

Comment:
@Kreyren The custom folder has been put here as an example, reference for people who want to use their custom aliases/completion/plugins/themes.

If you are going to remove it, please also documented regards the "custom" in Wiki.

Thanks.

Reply:
@nntoan Said directory and files are not removed, but renamed on `example.aliases.bash`